packages/modular-scripts/src/addPackage.ts

```diff
@@ -181,11 +187,13 @@ async function addPackage({
     }
 
     await templateInstallSubprocess;
-    newModularPackageJsonPath = require.resolve(installedPackageJsonPath);
+    templatePackageJsonPath = require.resolve(installedPackageJsonPath, {
+      paths: [modularRoot],
```

Comment:
When calling modular scripts from another modular repository it Require.resolve would resolve in the repo holding the script, instead of what it's run on.

Changed require.resolve() to always check from the Modular Root as identified by modular, which is the repo modular is run on, not from.

Curious to see if anyone has any concerns about this change, I feel like it's actually an improvement that will provide more consistent behaviour, and only really affect tests/us devs when we want to test modular changes.

Reply:
I don't see any evident downside to this. 👍